examples/angular_view_engine/src/BUILD.bazel

```diff
@@ -83,23 +84,22 @@ _ASSETS = [
     "@npm//:node_modules/zone.js/dist/zone.min.js",
 ]
 
+inject_html(
+    name = "inject_scripts_for_dev",
+    src = "//src:example/index.html",
```

Comment:
nit: make the input file`//src:example/index.dev.html` and output to `src/example/index.html` so the distinction between dev & prod is clearer in the sources?

Reply:
I think it's nice that at least one of the files has the same path in the srcs and the package.

Ideally I think we should have only `src/index.html` and devmode and prodmode both do the right thing - need to figure out diff. loading again, maybe using the manifest file with hinting, Jan can probably help us get it right

Reply:
That's a good point. It seems to me that any auto-detection for diff loading will always be broken for some use cases out there. Maybe the correct shape is to have inject_html have different srcs attributes for css, plain scripts, scripts with `nomodule` and scripts with `type="module"`.

However, probably not worth the effort or complexity at this point as the proper solution is to have an stand-alone npm package that can handle those distinctions and user's would just use the generated `npm_package_bin`.

Reply:
There is no way for an injector to know that you'd want `nomodule` on scripts such as
```    <script nomodule src="/core.min.js"></script>
    <script nomodule src="/system.js"></script>
```
for example